Assets/HoloToolkit/Input/Scripts/GestureManager.cs

```diff
@@ -251,5 +284,26 @@ void OnDestroy()
             InteractionManager.SourceUpdated -= InteractionManager_SourceUpdated;
             InteractionManager.SourceLost -= InteractionManager_SourceLost;
         }
+
+        public void OnSelect()
```

Comment:
oh, okay, the git review thing confused me, I thought this code was in GestureManager.